SIL.Windows.Forms.WritingSystems/LanguageLookupControl.cs

```diff
@@ -247,5 +247,11 @@ private void _searchText_TextChanged(object sender, EventArgs e)
 		{
 			_model.SearchText = _searchText.Text;
 		}
+
+		private void _showDialectsCheckBox_CheckedChanged(object sender, EventArgs e)
```

Comment:
For WeSay And Bloom, I don't want to have a new checkbox. Those apps just don't want this added complexity at all.  If those are the only apps that use this, it maybe be easier for you to just hard code this filter until some app comes along that want it.